crates/cheatcodes/src/inspector.rs

```diff
@@ -926,6 +931,13 @@ impl Cheatcodes {
 
         None
     }
+
+    pub fn rng(&mut self) -> &mut impl Rng {
+        self.rng.get_or_insert(match self.config.seed {
+            Some(seed) => StdRng::from_seed(seed.to_be_bytes::<32>()),
+            None => StdRng::from_entropy(),
+        })
```

Comment:
```suggestion
        self.rng.get_or_insert_with(|| match self.config.seed {
            Some(seed) => StdRng::from_seed(seed.to_be_bytes::<32>()),
            None => StdRng::from_entropy(),
        })
```

Reply:
When running `cargo clippy`, I got this warning:

```bash
warning: try not to call a closure in the expression where it is declared
   --> crates/cheatcodes/src/inspector.rs:936:32
    |
936 |           self.rng.get_or_insert((|| match self.config.seed {
    |  ________________________________^
937 | |             Some(seed) => StdRng::from_seed(seed.to_be_bytes::<32>()),
938 | |             None => StdRng::from_entropy(),
939 | |         })())
    | |____________^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_closure_call
    = note: `#[warn(clippy::redundant_closure_call)]` on by default
help: try doing something like
    |
936 ~         self.rng.get_or_insert((match self.config.seed {
937 +             Some(seed) => StdRng::from_seed(seed.to_be_bytes::<32>()),
938 +             None => StdRng::from_entropy(),
939 ~         }))
    |
```

Reply:
It's not the same: `get_or_insert` takes a value, this is computed always (eager); `get_or_insert_with takes a closure which is invoked only when `None` (lazy). Clippy is right because you're calling the closure yourself, when you want to pass it to the function. The code suggestion is what you want to do

Reply:
We could do this also, what do you think?

```rust
pub fn rng(&mut self) -> &mut impl Rng {
        let new_rng = match self.config.seed {
            Some(seed) => StdRng::from_seed(seed.to_be_bytes::<32>()),
            None => StdRng::from_entropy(),
        };
        self.rng.get_or_insert((|| new_rng)())
    }
```

Reply:
That's the same mistake. Please just apply the suggestion :)

Reply:
Lol totally missed the `_with`, it's getting late...